Make option values optional, defaulting to true (#502)
unicode-org/message-format-wg: spec/syntax.md

An **_<dfn>option</dfn>_** is a key-value pair
containing a named argument that is passed to a _function_.

An _option_ has a _name_ and optionally a _value_.

> An _option_ without a _value_ is taken to have the literal value `true`.

Reviewer: "Is taken" is a little bit elliptical.

```suggestion
An _option_ without a _value_ has the literal value `true`.
```
